core/src/main/java/org/apache/spark/shuffle/sort/ShuffleExternalSorter.java

```diff
@@ -82,6 +82,9 @@
   /** The buffer size to use when writing spills using DiskBlockObjectWriter */
   private final int fileBufferSizeBytes;
 
+  /** The buffer size to use when writes the sorted records to an on-disk file */
```

Comment:
@jiangxb1987 
thanks for review it,
The UnsafeSorterSpillWriter changes were updated.
please review it.